rest/debug.go

```diff
@@ -85,6 +88,12 @@ func (g *goroutineTracker) recordSnapshot() {
 
 	// append to history
 	g.Snapshots = append(g.Snapshots, numGoroutines)
+
+	// drop the last one if we've gone over the max
```

Comment:
The comment is slightly confusing since its dropping the item at slice index 0, perhaps change 'last' to 'oldest'